### PR TITLE
Interests save for guest user signup

### PR DESCRIPTION
**Pull request by divincoder**
Description:
- added PlaceInterests to user models and DB models
- updated BaseUrls for event and places
- Saved interests to sharedPref for guest flow and automatically save during signup

### CODE REVIEW

**Comment by divincoder** [comment]
@asogwachiuba review this PR

File: lib/features/signup/views/select_place_interest/select_place_interests_viewmodel.dart [hunk 1]
```diff
@@ -156,14 +154,4 @@ class SelectPlaceInterestViewModel extends IbloovViewModel {
   void onSkip() {
     navigationService.navigateTo(Routes.guestUserHomeView);
   }
-
```
Comment (asogwachiuba) [inline lib/features/signup/views/select_place_interest/select_place_interests_viewmodel.dart:159]: This refferal modal was a task from last sprint. Hope removing this is okay or was there something wrong with the implementation you noticed @divincoder.

File: lib/features/signup/views/select_place_interest/select_place_interests_viewmodel.dart [hunk 2]
```diff
@@ -108,9 +107,8 @@ class SelectPlaceInterestViewModel extends IbloovViewModel {
 
     setBusyForObject(savingInterest, true);
     final interestsList = _selectedInterests.map((e) => e.id ?? "").toList();
-    final referredBy = referredByController.text;
 
-    final body = {"interests": interestsList, "referredBy": referredBy};
```
Comment (asogwachiuba) [inline lib/features/signup/views/select_place_interest/select_place_interests_viewmodel.dart:113]: This reffered by was a recent update from last sprint. Please why is it been removed.

**Review by asogwachiuba — APPROVED** [review]

File: lib/features/signup/views/select_place_interest/select_place_interests_viewmodel.dart (hunk 1 above)
Comment (divincoder) [reply lib/features/signup/views/select_place_interest/select_place_interests_viewmodel.dart:159]: This is an old implementation. Did you do modal

File: lib/features/signup/views/select_place_interest/select_place_interests_viewmodel.dart (hunk 2 above)
Comment (divincoder) [reply lib/features/signup/views/select_place_interest/select_place_interests_viewmodel.dart:113]: This is an old implementation. We no longer register referredBy here

**Pull request merged**